elide-core/src/main/java/com/yahoo/elide/core/security/RequestScope.java

```diff
@@ -13,4 +16,5 @@
     String getApiVersion();
     String getRequestHeaderByName(String headerName);
     String getBaseUrlEndPoint();
+    Optional<MultivaluedMap<String, String>> getQueryParams();
```

Comment:
Let's make this Map<String, List<String>> so we can remove the dependency on javax.ws.rs.core from this interface.

Also - what about an empty map denoting optional?

Reply:
I dropped the Optional throughout the code. Hope that's what you had in mind.